docs/limitations.md

```diff
@@ -36,3 +36,49 @@ CSI ephemeral volumes may leave orphaned logical volumes
 
 The logical volume created by CSI ephemeral volumes may be left behind by restarting the node.
 This problem is because the kubelet on the restarted node may fail to remove the logical volume through the CSI driver when pods are removed.
+
+Use lvcreate-options at your own risk
+-------------------------------------------
+
+TopoLVM does not check the `lvcreate-options` that can optionally be added to a device-class.
+Therefore it cannot take them into consideration when scheduling, or do sanity checks for them.
+It is up to the user to make sure that these arguments make sense and work with the VG in question.
+For example, with `--type=raid1`, the VG must have at least 2 PVs to be able to create any LVs.
+
+Note also that the options may affect the "actual" available capacity.
+With `--type=raid1`, each LV will take up twice the normal space.
+You may want to tweak the `spare-gb` setting to avoid some issues with this.
+
+**Example**
+
+There is one VG `raid-vg` with two PVs (`disk1` and `disk2`).
+Then we create the following lvmd config:
+
+```yaml
+device-classes:
+  - name: "raid1"
+    volume-group: "raid-vg"
+    spare-gb: 100
+    lvcreate-options:
+      - "--type=raid1"
+```
+
+If we now ask for a volume with this device class it will be created using `lvcreate` with `--type=raid1`.
+This means that the data will be mirrored on the two disks.
+Notice the following:
+
+1. The VG *must* have at least two PVs in it with enough capacity or volume creation will fail.
+   This is a requirement coming from the RAID configuration and is up to the user to take into account when creating the VG and device-class.
+2. Since the data is mirrored on the two disks, it takes up twice as much space.
+   If we ask for a volume with 1 GB capacity, it will use 1 GB on each disk, i.e. 2 GB total of the VG.
+   TopoLVM does not know about this so it will not be considered when doing scheduling decisions.
+
+To help with the scheduling we can set `spare-gb` to the size of one disk.
+For example, if `disk1` and `disk2` are 100 GB each, we can set `space-gb` to 100 GB so that the reported capacity from TopoLVM would be 100 GB.
```

Comment:
```suggestion
For example, if `disk1` and `disk2` are 100 GB each, we can set `spare-gb` to 100 GB so that the reported capacity from TopoLVM would be 100 GB.
```

Reply:
Thanks, fixed!